### PR TITLE
Add survey guidance URL property to schema

### PR DESCRIPTION
**Pull request by jchapman68**
### What is the context of this PR?

A change to Survey Runner adding an external guidance link url necessitated a change to Validator.

### How to Review

Pull branch and build docker image for Schema Validator
Run Survey Runner and Schema Validator
Use docker ps in terminal to retrieve port number used against local host for Validator
Open REST Client
Set Method to POST
Set Address to http://localhost:[PORT]/validate
Set Body to mbs_0111.json from Survey Runner
Execute Send to confirm response is **200 OK**

Make url in Body invalid, eg `https` -> `hps` 
Execute Send to confirm response is **400 Bad Request**

### CODE REVIEW

**Comment by jchapman68** [comment]
[Guidance links on all live surveys](https://trello.com/c/csQmUQiQ/955-guidance-links-on-all-live-surveys)

File: schemas/common_definitions.json [hunk 1]
```diff
@@ -474,6 +474,10 @@
         "value"
       ]
     }
+  },
+  "url": {
+    "type": "string",
+    "pattern": "^(http:\/\/www\\.|https:\/\/www\\.|http:\/\/|https:\/\/)?[a-z0-9]+([\\-\\.]{1}[a-z0-9]+)*\\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$"
```
Comment (tgandrews) [inline schemas/common_definitions.json]: This seems really complex. Could we get away with something simpler like `http[s]?://\w+`?

Comment (samiwel) [reply schemas/common_definitions.json]: I've simplified it somewhat. Regex taken from https://www.regextester.com/94502

**Comment by samiwel** [comment]
I'm going to dismiss my own review because I've worked on the code change. We'll get someone else to review next week.

**Pull request merged**